libraries/tensorflow-gpu-1.12/install.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+# Install the requirements defined in the requirements.txt file in the template
+pip install -r context/requirements.txt
```

Comment:
We can change this to `pip3` if needed.

Reply:
yeah i think this will be okay to test and we'll know pretty fast whether it needs to be pip3 (but that would also mean that we would need a separate package to install tensorflow-1.12 for python2 too)

Reply:
Cool. We can change this easily.
We can add new a new package for python2 after testing for this is done.

Reply:
Also, if we're going to create a new package for python2, should we also include `py2` in the name of the packageset?